types/typeof/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for typeof 1.0
+// Project: https://github.com/CodingFu/typeof
+// Definitions by: OpenByteDev <https://github.com/OpenByteDev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function typeOf(object: any): string;
```

Comment:
~Why not `"string" | "number" | "array" | "null" | "buffer"~ ah nevermind, this does funky stuff on class instances.